dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentCDIApp/src/concurrent/mp/fat/cdi/web/MPConcurrentCDITestServlet.java

```diff
@@ -69,6 +69,15 @@
     @Inject
     BeanManager beanManager;
 
+    @Inject
+    RequestScopedBean requestBean;
+
+    @Inject
+    SessionScopedBean sessionBean;
+
+    @Inject
+    ConversationScopeBean conversationBean;
+
```

Comment:
I suggest you add ApplicationScoped and Dependent Scoped bean to this test app.

Reply:
adding a Dependent scoped bean is a good idea -- I'll add that. For ApplicationScoped it is just a no-op because application scoped context is already available on all threads